src/index.css

```diff
@@ -9073,6 +9073,10 @@ a:hover {
   color: #742a2a;
 }
 
+.text-orange {
```

Comment:
Just trying to work out why *removing* this class from footer added it, think it's because it's used on the not found page too. So think it's all cool, can I just check that you generated this file with tailwind-gen rather than add manually? Looks like that's the case just want to check, I should prob update README on this also.

Reply:
Hi, I think the text-orange class was added in by the other guy who first tried to contribute to this. As I forked from his, I found out that the text was orange which is different from the original version. I simply removed the class from the html.

Reply:
Also, speaking of running node commands, the 'npm start' didn't work for me as it uses semicolon. I had to change it to use '&' instead. @jenkoian Of course I didn't commit the package.json file as I feared I am the only one experiencing this issue.

Reply:
What OS you on?

Reply:
Working on Windows 10

Reply:
ah poss a windows thing then, will try and take a look if I get chance

Reply:
Great contributing to this project